CRM/Odoosync/Upgrader.php

```diff
@@ -119,4 +129,86 @@ private function getDefaultOptionValue($optionGroupName) {
     return (int) $value['values'][0]['value'];
   }
 
+  /**
+   * Install default message template
```

Comment:
after renaming the method below, change this description to : 

Creates  'CiviCRM-Odoo Sync Error Report' message template